ldap/servers/slapd/plugin_syntax.c

```diff
@@ -500,16 +528,44 @@ slapi_mods_syntax_check(
                 /* Loop through the values and validate each one */
                 for (j = 0; mod->mod_bvalues[j] != NULL; j++) {
                     if (syntax_plugin->plg_syntax_validate(mod->mod_bvalues[j]) != 0) {
+                        char invalid_val[INVALID_BUFSIZ] = {0};
+                        char buffer[BUFSIZ];
+                        if (mod->mod_bvalues[j]->bv_len > 0) {
+                            if (mod->mod_bvalues[j]->bv_len >= INVALID_BUFSIZ) {
+                                if (mod->mod_bvalues[j]->bv_val[mod->mod_bvalues[j]->bv_len] != '\0') {
+                                    PR_snprintf(invalid_val, INVALID_BUFSIZ - 3,
```

Comment:
why -3 ?

Reply:
To leave room for "..." .  This is how we do it in other places.

Anyway this is turning into a can of worms and I'm not sure its worth it anymore.   I have other more important things that need to get done so I'm going to shelf this PR for now...